cost-model/src/cost_model.rs

```diff
@@ -104,33 +100,35 @@ impl CostModel {
             );
     }
 
-    fn get_writable_accounts(transaction: &SanitizedTransaction) -> Vec<Pubkey> {
+    pub fn writable_accounts_iter(
```

Comment:
not ideal place for this function to live. Will, hopefully, put this somewhere shared after this is resolved: https://github.com/anza-xyz/agave/pull/1918#issuecomment-2206723634


While this iteration is not as fast as simply iterating over a Vec; I do not think it will be so much slower that we reduce overall performance.